security/pkg/server/grpc/authorizer.go

```diff
@@ -47,3 +49,26 @@ func (authZ *sameIDAuthorizer) authorize(requester *caller, requestedIDs []strin
 
 	return nil
 }
+
+// registryAuthorizor uses an underlying identity registry to make authorization decisions
+type registryAuthorizor struct {
```

Comment:
So there will be another PR to hook up the Control Plane with this registryAuthorizor, right?

Reply:
I'm not sure whether to add a cmd option to let user select which authorizor to use, or just switch to `registryAuthorizor` by default (for this I need to ensure that existing mesh expansion script works fine).

Reply:
The current one is pretty dummy and we should replace it. It would be great to provide an option for user to select once we have more complete authorizors. :)

Reply:
Can be done in a follow up PR :)